tests/fixture_test.go

```diff
@@ -171,11 +171,7 @@ func (fixture *Fixture) OneCallBuyShipment() *easypost.Shipment {
 // If you need to re-record cassettes, increment the date below and ensure it is one day in the future,
 // USPS only does "next-day" pickups including Saturday but not Sunday or Holidays.
 func (fixture *Fixture) BasicPickup() *easypost.Pickup {
-<<<<<<< HEAD
-	pickupDate := time.Date(2022, time.June, 27, 0, 0, 0, 0, time.UTC)
```

Comment:
@jchen293 I think you deleted the wrong pickupDate here, the one that's left is in the past and is probably causing the cassettes/tests to fail

Reply:
Well, the new cassettes passed, somehow...

Reply:
I forgot to save the files after rebasing before pushing the code which was causing the CI build to fail.